src/create/aws/create-router-deployments/index.js

```diff
@@ -60,7 +60,7 @@ module.exports = function createRouters(params, callback) {
         else {
           stop()
           if (urls.length === 2) {
-            console.log(chalk.green.dim('✓'), chalk.cyan.dim('Successfully deployed'))
+            console.log(chalk.green.dim('✓'), chalk.cyan.dim('Deployed'))
```

Comment:
Super nit 🤓: funny, last night this log line made me do a double-take. I think the terminology used here has led / still leads me to confusion, as we say 'Deployed' during the `create` command meanwhile we have a separate `deploy` command as well.

Reply:
yeaaaaaa…maybe we should be extra boring/explicit:

`AWS resources successfully provisioned` (?) …

Reply:
yeah or keep it consistent: "AWS resources created" ?